System.Compression.Primitives/System/Interop/Interop.Brotli.cs

```diff
@@ -0,0 +1,79 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Runtime.InteropServices;
+using System.IO.Compression;
+
+
+#if BIT64
+    using nuint = System.UInt64;
+#else 
+    using nuint = System.UInt32;
+#endif 
+
+namespace System.IO.Compression
+{
+    internal static partial class Interop
```

Comment:
Because realization for Unix system may be different like here https://github.com/dotnet/corefx/tree/master/src/System.IO.Compression/src/Interop